conwayste/src/ui/textfield.rs

```diff
@@ -151,19 +149,21 @@ impl TextField {
     pub fn clear(&mut self) {
         self.text.clear();
         self.cursor_index = 0;
+        self.visible_start_index = 0;
+        self.visible_end_index = 0;
         self.blink_timestamp = None;
         self.draw_cursor = false;
     }
 
-    /// Advances the cursor to the right by one
-    pub fn inc_cursor_pos(&mut self) {
+    /// Advances the cursor to the right by one character
```

Comment:
To me, any of the words advance/increment/decrement are confusing when talking about moving the cursor left or right. Just call it move left or move right.